auraed/src/vms/virtual_machine.rs

```diff
@@ -98,13 +102,15 @@ impl From<VmSpec> for vmm::vm_config::VmConfig {
             pmem: None,
             serial: default_serial(),
             console: default_console(),
+            #[cfg(target_arch = "x86_64")]
             debug_console: DebugConsoleConfig::default(),
```

Comment:
it makes me grumpy that we need to essentially copy the same constraints as are in upstream, because this means a future maintenance headache as fields are added or constraints are changed (what happens when upstream supports arm? how do we know to include it now?) but i guess it's Rust WAI.